Linguini.Bundle.Test/Linguini.Bundle.Test.csproj

```diff
@@ -24,4 +24,10 @@
         <AdditionalFiles Include="..\PluralRules.Generator\cldr_data\plurals.xml" />
     </ItemGroup>
 
+    <ItemGroup>
+        <PackageReference Include="IndexRange" Version="1.0.2" PrivateAssets="all" />
```

Comment:
Honestly I'd use [conditional package references](https://markheath.net/post/csproj-conditional-references)

I.e. 
```xml
<PackageReference Condition=" '$(TargetFramework)' == 'net462' "Include="IndexRange" Version="1.0.2" PrivateAssets="all"  />
```

Or putting it on whole ItemGroup